### DISCUSSION TITLE
Integrated values for plots

### DISCUSSION THREAD

**Discussion by plasim** [google-deepmind/torax#135] [Ideas]
Thanks for this nice framework for fusion! A small but nice feature in plots (--plot_progress) would be to see the volume integrated heating and total current values. That would be informative and also make it easier to see when there are issues with e.g. simulation setup and sources.

**Comment by jcitrin** [comment]
Thanks for the feedback! 

Fully agree about this feature. It's on the short-term roadmap both for live plotting, as well as in the post-run simple analysis tool (torax/plotting/plotruns.py). Can include e.g. integrated heating, fusion power, current, bootstrap current fraction, Greenwald fraction. Various integrated quantities can also be included as 0D post-processed output. Will keep you updated on the progress.
